cxx-squid/src/test/resources/metrics/public_api.h

```diff
@@ -45,6 +45,9 @@ class testClass
 	A(A const&) = delete;
 
 	// ignore defaulted methods
+  /**
+  operator= doc
+  */
```

Comment:
I think there is an issue in AbstractCxxPublicApiVisitor and the "operator=" is now effective because of my grammar updates but not handled correctly. I wanted to get some hints from @lcintrat but did not get feedback until now.
After my investigation for the issue I have some concerns about the design of this visitor and in my understanding most the methods should be implemented in *cxx-checks* and not in *cxx-squid*. The class has a strong dependency to the AST which creates maintenance effort for further grammar updates.